### PR TITLE
Fill changelog for v0.18.

### PR DESCRIPTION
**Pull request by adonath**
**Description**

This PR fills the changelog for Gammapy v0.18. I've summarised the most important bug fixes, API changes and new features.
I'll fill the list with the relevant PRs tomorrow.

**Dear reviewer**
Pleas do a quick review @registerrier, @AtreyeeS, @luca-giunti, @Bultako

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,7 +1,102 @@
...
+- 160 pull requests (not all listed below)
+
+**What's new**
+
+Gammapy v0.18 comes with many new features, further unified API, bug fixes
+and improved performance.
+
+For Gammapy v0.18 the handling of the instrumental background was refactored
+by introducing a ``FoVBackgroundModel`` which is specific to each ``MapDataset``.
+Instead of relying on the previous ``BackgroundModel``, which couples
```
Comment (AtreyeeS) [inline CHANGES.rst:15]: which coupled*

File: CHANGES.rst [hunk 2]
```diff
@@ -1,7 +1,102 @@
...
+``PiecewiseNormSectralModel``. The EBL absorption was refactored
+accordingly to an ``EBLAbsorptionNormSpectralModel``. A new
+``GeneralizedGaussianSpatialModel`` has been introduced. Gammapy v0.18 comes
+now with support for custom energy dependent spatial models. The ``SkyDiffuseCube``
+had been removed, the same functionality can now be achieved with the ``TemplateSpatialModel``.
+Following the proposal in `PIG 21`_, short YAML tags were introduced for
+all models. An overview of the tags can be found in a table in the linked
+PIG document.
+
+Gammpy v0.18 now supports an ``EDispKernelMap``, which stores a spatially
```
Comment (AtreyeeS) [inline CHANGES.rst:53]: `EdispKernelMap` was supported in v0.17 as well - but the stacking was done incorrectly [The mask was applied incorrectly, throwing away a lot of data]. This has now been corrected

File: CHANGES.rst [hunk 3]
```diff
@@ -1,7 +1,102 @@
...
+dependent energy redistribution matrix. This allows to handle the safe
+energy threshold for stacked analyses correctly. The ``MapDataset.edisp``
+attribute has been changed to this class by default. The ``IRFStacker`` class has been removed.
+
+All ``Estimator`` classes support now the definition of energy bins on init. This
+means light-curves, excess maps, excess profiles and ts maps can be computed in
+energy bins with the same API.
+
+An ``Fit.stat_surface()`` method was introduce which allows to compute a
```
Comment (AtreyeeS) [inline CHANGES.rst:62]: A not an

File: CHANGES.rst [hunk 4]
```diff
@@ -1,7 +1,102 @@
...
+
+- Atreyee Sinha
+- Axel Donath
+- Fabio Pintore
+- José Enrique Ruiz
+- Luca Giunti
+- Quentin Remy
+- Régis Terrier
+- Laura Olivera Nieto
+- Alexis de Almeida Coutinho
```
Comment (AtreyeeS) [inline CHANGES.rst:87]: This violates alphabetic order - Alexis should be first and Laura comes before Luca

**Review by AtreyeeS — APPROVED** [review]
Thanks @adonath ! Looks great, left some minor inline changes

Maybe you can also add
- Naming conventions (specially energy) have been harmonised
- Improved functionality for handling WStat statistics

File: CHANGES.rst [hunk 5]
```diff
@@ -1,7 +1,102 @@
...
+dependent energy redistribution matrix. This allows to handle the safe
+energy threshold for stacked analyses correctly. The ``MapDataset.edisp``
+attribute has been changed to this class by default. The ``IRFStacker`` class has been removed.
+
+All ``Estimator`` classes support now the definition of energy bins on init. This
+means light-curves, excess maps, excess profiles and ts maps can be computed in
+energy bins with the same API.
+
+An ``Fit.stat_surface()`` method was introduce which allows to compute a
+fit statistic surface. In addition am option to store the trace of the fit was
```
Comment (AtreyeeS) [inline CHANGES.rst:63]: an* option

File: CHANGES.rst (hunk 4 above)
Comment (registerrier) [reply CHANGES.rst:87]: The list misses:
Bruno Khelifi
Cosimo Nigro
Fabio Acero
Lea Jouvin

File: CHANGES.rst [hunk 6]
```diff
@@ -1,7 +1,102 @@
...
+Following the proposal in `PIG 21`_, short YAML tags were introduced for
+all models. An overview of the tags can be found in a table in the linked
+PIG document.
+
+Gammpy v0.18 now supports an ``EDispKernelMap``, which stores a spatially
+dependent energy redistribution matrix. This allows to handle the safe
+energy threshold for stacked analyses correctly. The ``MapDataset.edisp``
+attribute has been changed to this class by default. The ``IRFStacker`` class has been removed.
+
+All ``Estimator`` classes support now the definition of energy bins on init. This
```
[the remark this answers is not in the document]
Comment (registerrier) [reply CHANGES.rst:58]: Maybe a sentence on:
The `Estimator` API has been homogenized. In particular, they now support the definition of...

**Review by registerrier — DISMISSED** [review]
Thanks @adonath . Some small comments inline

File: CHANGES.rst [hunk 7]
```diff
@@ -1,7 +1,102 @@
...
+
+Gammapy v0.18 comes with many new features, further unified API, bug fixes
+and improved performance.
+
+For Gammapy v0.18 the handling of the instrumental background was refactored
+by introducing a ``FoVBackgroundModel`` which is specific to each ``MapDataset``.
+Instead of relying on the previous ``BackgroundModel``, which couples
+the map template and spectral parameters, the information is now split such that
+``MapDataset.background`` contains the map template and the ``FoVBackgroundModel``
+the corresponding parameters. The model corrected background can now be
```
Comment (fabiopintore) [inline CHANGES.rst:18]: the information is now split such that ``MapDataset.background`` and ``FoVBackgroundModel`` contain the map template and the corresponding parameters, respectively.

File: CHANGES.rst (hunk 3 above)
Comment (fabiopintore) [inline CHANGES.rst:62]: was introduced

File: CHANGES.rst (hunk 4 above)
Comment (registerrier) [reply CHANGES.rst:87]: Also
Jalel Hajlaoui

**Comment by adonath** [comment]
Thanks @registerrier, @AtreyeeS and @fabiopintore for the feedback. I implemented all proposed changes and also added the list of PRs.

**Pull request merged**